packages/babel-helper-create-class-features-plugin/src/fields.js

```diff
@@ -429,7 +471,47 @@ function buildPublicFieldInitSpec(ref, prop, state) {
 }
 
 function buildPrivateStaticMethodInitLoose(ref, prop, state, privateNamesMap) {
-  const { id, methodId } = privateNamesMap.get(prop.node.key.id.name);
+  const privateName = privateNamesMap.get(prop.node.key.id.name);
+  const { id, methodId, getId, setId, initAdded } = privateName;
+  if (initAdded) return;
```

Comment:
Why do we need guard against double entrance? IMO this function should run once for every `prop`.